int32 loadcell0
int32 loadcell1